test/job.js

```diff
@@ -13,15 +13,17 @@ const Job = require('../lib/job');
 
 const mongoHost = process.env.MONGODB_HOST || 'localhost';
 const mongoPort = process.env.MONGODB_PORT || '27017';
-const mongoCfg = 'mongodb://' + mongoHost + ':' + mongoPort + '/agenda-test';
+const agendaDatabase = 'agenda-test';
+const mongoCfg = 'mongodb://' + mongoHost + ':' + mongoPort + '/' + agendaDatabase;
 
 // Create agenda instances
 let agenda = null;
-let mongo = null;
+let mongoDb = null;
+let mongoClient = null;
 
 const clearJobs = () => {
   return new Promise(resolve => {
-    mongo.collection('agendaJobs').remove({}, resolve);
+    mongoDb.collection('agendaJobs').deleteMany({}, resolve);
```

Comment:
https://docs.mongodb.com/manual/reference/method/db.collection.deleteMany/

Reply:
Pretty sure mongo client returns promises if you don't pass a callback, then you'd also get the rejection, rather than resolving with error|null

http://mongodb.github.io/node-mongodb-native/3.0/api/Collection.html#~deleteWriteOpCallback

Reply:
Hum, I'm getting unexpected test failure with this change:

![image](https://user-images.githubusercontent.com/87168/41402856-0f78fc5a-6fcc-11e8-8a06-4e4fd7397fa8.png)